data/human/free worlds 1 start.txt

```diff
@@ -1034,7 +1034,7 @@ mission "FW Senate 1B"
 				`	"I have no idea."`
 				`	"Ummm... they both involve momentous decisions that have serious consequences?"`
 				`	"They both require someone skilled and decisive to be in command?"`
-			`	"The correct answer," she says, "was, 'nothing whatsoever; they are entirely different disciplines.' Now, hurry up, I hope to be on Bourne a week from now." You consider explaining to her that it's impossible to get there that fast, but decide against it.`
+			`	"The correct answer," she says, "was, 'nothing whatsoever; they are entirely different disciplines.' Now, hurry up, I hope to be on Bourne a week from now." You consider explaining to her that it's unreasonable to get there that fast using normal hyperlanes, but decide against it.`
```

Comment:
This is a good start, but if the player hasn't encountered jump drives, then they're not going to be thinking in terms of "normal" hyperlanes since they don't know there could be any other type.

Perhaps "You consider explaining to her that the hyperlanes simply don't allow you to get there that fast"?